src/components/RepoCard.astro

```diff
@@ -0,0 +1,157 @@
+---
+import { Icon } from "astro-icon"
+
+export type Props = {
+	title: string
+	sourceUrl: string
+	stackblitzUrl: string
+	codesandboxUrl: string
+	gitpodUrl: string
+	previewImage: string | null
+}
+
+const {
+	title,
+	sourceUrl,
+	stackblitzUrl,
+	codesandboxUrl,
+	gitpodUrl,
+	previewImage,
+} = Astro.props
+---
+
+<article class="relative border border-astro-gray-500 bg-astro-gray-600">
+	<div class="noise z-0"></div>
+	<div class="aspect-video bg-astro-gray-700">
+		{
+			!!previewImage && (
+				<img src={previewImage} alt="" class="object-cover s-full" />
```

Comment:
The screenshot here is getting cut off at the top. Should we be aligning the image to the top of the container to avoid that?

<img width="417" alt="image" src="https://user-images.githubusercontent.com/357379/230943824-420797a6-a6bb-4db6-92b0-c251b0a9552b.png">

Reply:
Similarly some of our screenshots seem off. Do we need to redo these so they have the right aspect ratio?

<img width="422" alt="image" src="https://user-images.githubusercontent.com/357379/230944091-cf328124-5f52-49f3-8666-82a9d2d589fd.png">

Reply:
+1, it's a little tedious but would be worth opening up each demo locally and getting a screenshot that matches aspect ratio

I don't think we have all these demos hosted live anywhere to grab screenshots from unfortunately

Reply:
I looked at what the current astro.new site does, and it just aligns them all to the top-left, so I did that here 😇